nested_lookup/lookup_api.py

```diff
@@ -30,27 +30,53 @@ def _nested_delete(document, key):
 
 
 def nested_update(document, key, value, in_place=False):
+    """
+    Method to update a key->value pair in a nested document
+    Args:
+        document: Might be List of Dicts (or) Dict of Lists (or)
+        Dict of List of Dicts etc...
+        key: Key to update the value
+    Return:
+        Returns a document that has updated key, value pair.
+    """
+
+    # check if a list or scalar value is provided and create a list from the scalar value
+    # check the length of the list and provide it to _nested_update
+    if type(value) == list:
```

Comment:
These lines can be changed to
`if not isinstance(value, list):
      value = [value]
  val_len = len(value)`